components/HeroHome.tsx

```diff
@@ -0,0 +1,46 @@
+"use client"
```

Comment:
I do like that at lease the `"use client"` is in smaller components than the `ClientLayout`. 

A clean up for later could be moving the "use client" to some version of the button component? No need to do that in this PR. This is certainly an improvement! 

https://sentry.io/answers/next-js-13-button-onclick-event-handlers-cannot-be-passed-to-client-component-props/